### DISCUSSION TITLE
Force update of all cells

### DISCUSSION THREAD

**Discussion by mark-0lchanski** [andmarti1424/sc-im#781] [Q&A]
Hey,

I have a cell that `@evaluate`s a string in another cell. As the docs state, the value does not update itself which is fine, but the problem is that its value is displayed as `0` when opening the file.
Is there a way to force the evaluation on start up, or any other way to update cells?

**Comment by andmarti1424** [comment]
Hello. Try `@`. If it does not work. please share your file.
